tests/integration/targets/iosxr_snmp_server/tests/common/empty_config.yaml

```diff
@@ -0,0 +1,50 @@
+---
+- debug:
+    msg: START iosxr_snmp_server empty_config integration tests on connection={{ ansible_connection }}
+
+- name: Merged with empty config should give appropriate error message
+  register: result
+  ignore_errors: true
+  cisco.iosxr.iosxr_snmp_server:
+    config:
+    state: merged
+
+- assert:
+    that:
+      - result.msg == 'value of config parameter must not be empty for state merged'
+
+- name: Replaced with empty config should give appropriate error message
+  register: result
+  ignore_errors: true
+  cisco.iosxr.iosxr_snmp_server:
+    config:
+    state: replaced
+
```

Comment:
Can you add a task for the state: "overridden"?

Reply:
ack